app/models/user.rb

```diff
@@ -716,6 +716,14 @@ def self.current
     @current_user ||= User.anonymous
   end
 
+  def self.execute_as(user)
```

Comment:
Ah, thanks. I missed that skimming through the templates. Makes sense, but if there's any time (left), we should strive to modify this.